chapters/inheritance.tex

```diff
@@ -115,6 +115,7 @@ \section{Inheritance -- Extends Clause}\label{inheritance-extends-clause}
 \subsection{Multiple Inheritance}\label{multiple-inheritance}
 
 Multiple inheritance is possible since multiple \lstinline!extends!-clauses can be present in a class.
+As stated in \cref{the-inherited-contents-of-the-element}, it is illegal for one of these \lstinline!extends!-clauses to influence the lookup of another's (or its own) class name.
```

Comment:
I don't understand the genitive in "another's". Isn't clearer to write the class name of any extends clause (including itself) or something like that.

Reply:
I ended up with this formulation because I wanted a (easy reading) reference to the `extends`-clauses in the same class – because this is what we mean, right?  Once I know the answer to this, I can try to reformulate.

Reply:
Yes, that's what we mean - so writing "the class name of another extends-clause" (I don't know how to add its own there) would make sense - it was just the "another's" that seems odd.

Reply:
To me, _another `extends`-clause_ has the risk of being interpreted as also including `extends`-clauses in other classes, such as in derived classes.  The _another's_ was meant to keep the reference to _one of these `extends`-clauses_ without the need to repeat that phrase.

Is this too verbose?
> … it is illegal for these `extends`-clauses to interfere by influencing the lookup of one another's class names (unrelated to multiple inheritance, an individual `extends`-clause is also forbidden to interfere with itself in this way).

Reply:
> To me, _another `extends`-clause_ has the risk of being interpreted as also including `extends`-clauses in other classes, such as in derived classes. The _another's_ was meant to keep the reference to _one of these `extends`-clauses_ without the need to repeat that phrase.
> 
> Is this too verbose?
> 
> > … it is illegal for these `extends`-clauses to interfere by influencing the lookup of one another's class names (unrelated to multiple inheritance, an individual `extends`-clause is also forbidden to interfere with itself in this way).

The main issue to me isn't verbosity, but that it isn't clear what "another" (or "another's") refers to.
Writing "another extends-clause" or "another extends-clause in the class" avoids any such issue.

However, the parenthesis seems too verbose and unneeded.
If we here want to clarify that it is illegal to influence the look of any other or extends-clause in the class and also the current extends-clause it is simpler to write "any extends-clause in the class" instead of "another"

Reply:
OK, I made an attempt in this direction.  Since the sentences of the paragraph now appeared more disconnected, I made it a non-normative paragraph instead.